tests/test_py_docstring.py

```diff
@@ -58,4 +58,9 @@ def test_real_input_file(self):
 
         # Just ensure that the bytestream is the appropriate length.
         # This is a terrible test. :-/
-        assert len(output_file.getvalue()) == 25294
+        assert len(output_file.getvalue()) == 25318
+
+    def test_init_files(self):
+        files = ['foo.proto', '/bar.proto', 'baz/qux/corge.proto']
+        expected = {'__init__.py', 'baz/qux/__init__.py'}
```

Comment:
Oh, this is odd. I know why it does this; we do not want to generate init files above the protos, but it is strange that in this case we generate the root and leaf, but not the branch.

Whatever. Leave as-is.

Reply:
Gotcha. 


> Leave as-is

👀 Nice pun 👍